client/src/lib/tests/ReactRouteRegister-test.js

```diff
@@ -31,7 +31,7 @@ describe('ReactRouteRegister', () => {
       expect(reactRouteRegister.getRootRoute())
         .toEqual({
           path: '/',
-          routes: jasmine.any(Function),
+          routes: expect.any(Function),
```

Comment:
jasmine isn't even included as a dependency, I have no idea how this test was passing in the past.